src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -49,3 +34,28 @@
     (utils/get-account-by-address accounts account-address)
     :balance
     (utils/get-balance-by-address balances account-address))))
+
+(defn- calc-token-value
+  [{:keys [symbol market-values-per-currency] :as item}]
+  (let [fiat-value                         (utils/sum-token-chains item)
+        market-values                      (:usd market-values-per-currency)
+        {:keys [price change-pct-24-hour]} market-values
+        fiat-change                        (* fiat-value
```

Comment:
I understand it's a simple function but should we also move this function
`(* fiat-value
                                              (/ change-pct-24-hour (+ 100 change-pct-24-hour)` to the utils and then add a test to cover it's behaviour. It's likely we need this in other screens/subs too 🤔

Reply:
@J-Son89 fixed ! 👍